docs/legal/CREDITS.md

```diff
@@ -203,12 +204,14 @@ Chocolatey uses [AlphaFS](https://github.com/alphaleonis/AlphaFS) for long file
   NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
   DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-~~~
+```
+
+### Checksum @ 0.2.0
 
-### Checksum
-Chocolatey uses [Checksum](https://github.com/ferventcoder/checksum) to determine checksums. [License terms](https://github.com/ferventcoder/checksum/blob/e6f5645610c7bc15084b48f69d4cdb056106f956/LICENSE):
+Chocolatey uses [Checksum](https://github.com/ferventcoder/checksum) to determine checksums.

```

Comment:
The checksum repository was moved to `https://github.com/chocolatey/checksum`. It's not super important to update since there is a redirect in place.

Reply:
@TheCakeIsNaOH thanks for catching this! I have fixed this up!